node/go.mod

```diff
@@ -240,3 +282,7 @@ require (
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 
 replace github.com/wormhole-foundation/wormhole/sdk => ../sdk
+
+replace github.com/CosmWasm/wasmd v0.28.0 => github.com/wormhole-foundation/wasmd v0.28.0-wormhole-2
+
+replace github.com/cosmos/cosmos-sdk => github.com/wormhole-foundation/cosmos-sdk v0.45.9-wormhole
```

Comment:
I think we do need these because in golang, we can't have different versions of a dependency.  I.e. having wormchain as a dep won't work while depending on a different, breaking version of cosmos-sdk.

Reply:
So I guess it's okay to close this? @jynnantonix please reopen if you disagree.